ddtrace/auto.py

```diff
@@ -19,3 +19,7 @@ def main():
 directly.
 """
 import ddtrace.bootstrap.sitecustomize  # noqa:F401
+from ddtrace.internal.telemetry import telemetry_writer
+
+
+telemetry_writer.add_configuration("ddtrace_auto_used", True, "unknown")
```

Comment:
Putting this in `auto.py` means that `import auto` falls out of sync with `import ddtrace.bootstrap.sitecustomize`. This isn't inherently a problem but it does change the meaning of `import auto` in a way that could surprise customers.

Reply:
I am a fan of adding telemetry configurations where operations occur. It makes the code easier to read. This module will now trigger `ddtrace.bootstrap.sitecustomize` AND collect telemetry on how this operation was executed.

Instead we could put this logic in `ddtrace.bootstrap.sitecustomize`. We can check if `ddtrace.auto` is in sys.modules and then queue this configuration.